discord_lib/discord_chat_service.py

```diff
@@ -15,11 +15,14 @@ def __init__(self, settings, client=None):
         """Accepts Django settings object and optional Discord APIClient (for testing)."""
         self._client = client or APIClient(settings.DISCORD_API_TOKEN)
         self._guild_id = settings.DISCORD_GUILD_ID
-        self._parent_name = settings.DISCORD_GUILD_CATEGORY
+        self._puzzle_category_name = settings.DISCORD_PUZZLE_CATEGORY
+        self._archived_category_name = settings.DISCORD_ARCHIVED_CATEGORY
```

Comment:
nit: Can we replace "ARCHIVED" with "ARCHIVE"? It seems to match "puzzle" as a noun and is one letter fewer.